Modules/CaPTkSurvival/include/CaPTkSurvival.h

```diff
@@ -0,0 +1,94 @@
+#ifndef CaPTkSurvival_h
+#define CaPTkSurvival_h
+
+// The following header file is generated by CMake and thus it's located in
+// the build directory. It provides an export macro for classes and functions
+// that you want to be part of the public interface of your module.
+#include <MitkCaPTkSurvivalExports.h>
+
+#include <QObject>
+#include <QFuture>
+#include <QFutureWatcher>
+#include <QProgressBar>
+
+namespace captk {
+/** \class CaPTkSurvival
+ *  \brief Survival Prediction Index Module API
+ */
+class MITKCAPTKSURVIVAL_EXPORT CaPTkSurvival : public QObject
+{
+    Q_OBJECT
+
+public:
+    CaPTkSurvival(QObject *parent = 0);
+
+    ~CaPTkSurvival() {}
+    QString cbicaModelDir;
```

Comment:
This should be a private or protected variable. Also "m_" is used as a prefix everywhere to signify private/protected variables. So this has to be changed m_CbicaModelDir (or m_CaPTkModelDir or m_ModelDir)

Reply:
Done, will update this PR momentarily